README.md

```diff
@@ -10,7 +10,22 @@ Our stretch goals include adding collaborators to any bucket list, viewing popul
 
 ## Project task requirements
 
-*placeholder*
+### Minimal requirements (will definitely complete)
+* User should be able to create, delete, edit buckets of travel destinations of interest
+* User should be able to add, remove, move, destinations within a bucket
+* All user information should persist in a database and retrieved upon site load
```

Comment:
Are we doing user info in minimal? Or should this be in standard.

Reply:
standard